app/services/doc_auth/mock/doc_auth_mock_client.rb

```diff
@@ -21,10 +22,16 @@ def self.mock_response!(method:, response:)
         @response_mocks[method.to_sym] = response
       end
 
+      def self.response_delay(delay)
```

Comment:
Adding delays in mock client to facilitate tests that reflect reality, useful for hybrid flow poller testing.

Reply:
This is a good idea. 👍🏻